### PR TITLE
post: update the foundation post

### PR DESCRIPTION
**Pull request by iGormilhit**
* Configures the date, publishDate and lastmod in order to diffentiate
  lastmod from the other dates.
* Documents in the README the lasmod usage.
* Update the theme in order to get the display of the lastmod parameter.

### CODE REVIEW

File: content/post/la-fondation-rero-est-nee.md [hunk 1]
```diff
@@ -1,56 +1,137 @@
...
+
+- __La gouvernance :__ RERO+ est désormais sous la supervision du Conseil de
+  fondation, composé au départ de :
+  - Damian Elsig, président jusqu'au 31 juillet
+    2021 (Canton du Valais)
+  - Elisabeth Baume-Schneider, présidente dès le 1er août 2021
+  - Antoine Grandjean, vice-président (République et Canton de Neuchâtel)
+  - Géraldine Rérat-Oeuvray (République et Canton du Jura)
+  - Thierry Petoud (Ville de Martigny)
+  - Martin Godd, consultant, ancien directeur de la Bibliothèque cantonale et
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:54]:
```suggestion
  - Martin Good, consultant, ancien directeur de la Bibliothèque cantonale et
```

File: content/post/la-fondation-rero-est-nee.de.md [hunk 2]
```diff
@@ -19,11 +20,12 @@ Angesichts dieser Veränderungen und mit Unterstützung der kantonalen Behörden
...
 - __Name:__ RERO war ursprünglich das Akronym für Réseau Romand, das später zum Réseau des bibliothèques de Suisse occidentale (Westschweizer Bibliotheksverbund) wurde. Heute ist die Institution, obwohl sie immer noch überwiegend französischsprachig ist, nicht mehr an territoriale Kriterien gebunden. Der Name RERO+ bedeutet eine Erweiterung ihrer Aktivitäten, ihrer Dienstleistungen sowie ihrer geografischen und sprachlichen Abdeckung, während sie gleichzeitig ihre Identität beibehält, die seit mehr als 30 Jahren in der Schweizer Informationslandschaft bekannt ist.
 
-- __Governance:__ RERO+ steht nun unter der Aufsicht des Stiftungsrates, der sich zunächst aus vier Vertretern der Gründungsmitglieder zusammensetzt:
-    - Damian Elsig, Präsident (Kanton Wallis)
+- __Governance:__ RERO+ steht nun unter der Aufsicht des Stiftungsrates, der sich zunächst zusammensetzt:
+    - Damian Elsig, Präsident bis 31. Juli 2021 (Kanton Wallis)
     - Antoine Grandjean, Vizepräsident (Kanton Neuenburg)
     - Géraldine Rérat-Oeuvray (Kanton Jura)
     - Thierry Petoud (Stadt Martigny)
+    - Martin Godde
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.de.md:28]:
```suggestion
    - Martin Good
```

File: content/post/la-fondation-rero-est-nee.en.md [hunk 3]
```diff
@@ -19,11 +20,13 @@ In view of these changes, and with the support of the cantonal authorities withi
...
 
-- __The governance:__ RERO+ is now under the supervision of the Foundation Board, initially composed of four representatives of the founding members:
-    - Damian Elsig, president (Canton of Valais)
+- __The governance:__ RERO+ is now under the supervision of the Foundation Board, initially composed of:
+    - Damian Elsig, president until July 31, 2021 (Canton of Valais)
+    - Elisabeth Baume-Schneider, president from August 1, 2021
     - Antoine Grandjean, vice-president (Canton of Neuchâtel)
     - Géraldine Rérat-Oeuvray (Canton of Jura)
     - Thierry Petoud (City of Martigny)
+    - Martin Godd, consultant, former director of the Canton library of Fribourg
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.en.md:29]:
```suggestion
    - Martin Good, consultant, former director of the Canton library of Fribourg
```

File: content/post/la-fondation-rero-est-nee.md [hunk 4]
```diff
@@ -1,56 +1,137 @@
...
+    universitaire de Fribourg
+
+  Au départ, chaque membre fondateur délègue un représentant au Conseil de
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:57]:
```suggestion
```

File: content/post/la-fondation-rero-est-nee.md [hunk 5]
```diff
@@ -1,56 +1,137 @@
...
+  fondation. Par la suite, ce principe ne devra pas forcément être respecté et
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:58]:
```suggestion
Par la suite, 
```

File: content/post/la-fondation-rero-est-nee.md [hunk 6]
```diff
@@ -1,56 +1,137 @@
...
+  avec plusieurs acteurs du monde des bibliothèques et de l'information en Suisse
+  et à l'étranger, notamment [SLSP](https://slsp.ch), la [Bibliothèque Nationale
+  Suisse](https://www.nb.admin.ch/), [l'ABES](https://abes.fr/) et la
+  [Bibliothèque nationale de France](https://www.bnf.fr/).
 
-- __Le mode de financement :__ RERO+ est financé exclusivement par la vente de ses services.
+---
 
-- __Les activités :__ Comme c’est le cas depuis environ deux ans, RERO+ va se concentrer sur le développement continu et la maintenance de ses services principaux : le système de gestion de bibliothèque de nouvelle génération RERO ILS et la gestion de contenus numériques avec SONAR. Ces logiciels et les services respectifs seront amenés à évoluer avec les besoins de sa clientèle, dont le cercle s’élargira à toute la Suisse.
+## Mot de Damian Elsig, président du Conseil de fondation RERO+
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:100]:
```suggestion
## Mot de Damian Elsig, président du Conseil de fondation
```

File: content/post/la-fondation-rero-est-nee.md [hunk 7]
```diff
@@ -1,56 +1,137 @@
...
 
 ---
 
-## Mot de Damian Elsig, président du Conseil de fondation RERO+
-
-La transformation de RERO vers RERO+ est de taille. La nouvelle orientation est le fruit d’une réflexion conjointe, sur la base d’une analyse de marché et de l’intérêt avéré pour les services sur mesure et/ou partagés des bibliothèques publiques, patrimoniales, scolaires et spécialisées en Suisse. RERO+ orientera ses prestations vers les besoins de ces types de bibliothèques, que ce soit par le développement de services ou par la mise à disposition de gestion informatique globale.
+## Nouvelle présidente dès le 1<sup>er</sup> août : Elisabeth Baume-Schneider
 
-L’équipe de RERO+ a son siège en Valais et est composée de spécialistes disposant d’une longue expérience dans ce domaine, avec la clientèle au centre de leur attention.
+Peu après la création de la fondation, Damian Elsig a été [nominé à la tête de
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:124]:
```suggestion
Peu après la création de la fondation, Damian Elsig a été [nommé à la tête de
```

File: content/post/la-fondation-rero-est-nee.md [hunk 8]
```diff
@@ -1,56 +1,137 @@
...
+Suisse](https://www.admin.ch/gov/fr/accueil/documentation/communiques.msg-id-83356.html).
+Une nouvelle personne intègrera le Conseil de fondation pour représenter le
+Canton du Valais.
 
-Les solutions informatiques sont développées en mode open source et se basent sur le logiciel [Invenio 3](https://inveniosoftware.org), développé par le CERN. Les données sont stockées en Suisse.
-
-RERO+ : une fondation à but non-lucratif estampillée *swiss made* !
+Quant à la présidence, elle sera assurée dès le mois d'août 2021 par [Élisabeth
+Baume-Schneider](https://fr.wikipedia.org/wiki/%C3%89lisabeth_Baume-Schneider),
+conseillère aux États, après la mise en production du nouveau système système
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:132]:
```suggestion
conseillère aux États.
```

File: content/post/la-fondation-rero-est-nee.md [hunk 9]
```diff
@@ -1,56 +1,137 @@
...
+RERO ILS : [bib.rero.ch](https://bib.rero.ch/).
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:133]:
```suggestion
```

File: content/post/la-fondation-rero-est-nee.md [hunk 10]
```diff
@@ -1,56 +1,137 @@
...
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:134]:
```suggestion

## Mise en production du nouveau système RERO ILS le 12 juillet

Facette la plus visible de cette grande transformation, l'ancien système de gestion de bibliothèque a été remplacé le 12 juillet par le nouveau RERO ILS, avec la mise en ligne du nouveau catalogue RERO+ : [bib.rero.ch](https://bib.rero.ch/).
```

**Review by mmo — CHANGES_REQUESTED** [review]

**Review by pronguen — APPROVED** [review]

File: content/post/la-fondation-rero-est-nee.md [hunk 11]
```diff
@@ -1,56 +1,142 @@
...
+
+- __La gouvernance :__ RERO+ est désormais sous la supervision du Conseil de
+  fondation, composé au départ de :
+  - Damian Elsig, président jusqu'au 31 juillet
+    2021 (Canton du Valais)
+  - Elisabeth Baume-Schneider, présidente dès le 1er août 2021
+  - Antoine Grandjean, vice-président (République et Canton de Neuchâtel)
+  - Géraldine Rérat-Oeuvray (République et Canton du Jura)
+  - Thierry Petoud (Ville de Martigny)
+  - Martin Good, consultant, ancien directeur de la Bibliothèque cantonale et
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.md:54]:
```suggestion
  - Martin Good, indépendant, ancien directeur de la Bibliothèque cantonale et
```

File: content/post/la-fondation-rero-est-nee.en.md [hunk 12]
```diff
@@ -1,31 +1,34 @@
...
-    - Damian Elsig, president (Canton of Valais)
-    - Antoine Grandjean, vice-president (Canton of Neuchâtel)
-    - Géraldine Rérat-Oeuvray (Canton of Jura)
+- __The governance:__ RERO+ is now under the supervision of the Foundation Board, initially composed of:
+    - Damian Elsig, president until July 31, 2021 (Canton of Valais)
+    - Elisabeth Baume-Schneider, president as of 1 August 2021
+    - Antoine Grandjean, vice-president (Republic and Canton of Neuchâtel)
+    - Géraldine Rérat-Oeuvray (Republic and Canton of Jura)
     - Thierry Petoud (City of Martigny)
+    - Martin Good, consultant, former director of the Canton and University Library of Fribourg
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.en.md:29]:
```suggestion
    - Martin Good, independent, former director of the Canton and University Library of Fribourg
```

File: content/post/la-fondation-rero-est-nee.de.md [hunk 13]
```diff
@@ -19,13 +20,15 @@ Angesichts dieser Veränderungen und mit Unterstützung der kantonalen Behörden
...
-    - Damian Elsig, Präsident (Kanton Wallis)
-    - Antoine Grandjean, Vizepräsident (Kanton Neuenburg)
-    - Géraldine Rérat-Oeuvray (Kanton Jura)
+- __Governance:__ RERO+ steht nun unter der Aufsicht des Stiftungsrates, der sich zunächst wie folgt zusammensetzt:
+    - Damian Elsig, Präsident bis 31. Juli 2021 (Kanton Wallis)
+	- Elisabeth Baume-Schneider, Präsidentin ab 1. August 2021
+    - Antoine Grandjean, Vizepräsident (Republik und Kanton Neuenburg)
+    - Géraldine Rérat-Oeuvray (Republik und Kanton Jura)
     - Thierry Petoud (Stadt Martigny)
+    - Martin Good, Berater, ehemaliger Direktor der Kantons- und Universitätsbibliothek Freiburg
```
Comment (mmo) [inline content/post/la-fondation-rero-est-nee.de.md:29]: «indépendant» (en DE), plutôt que Berater

**Review by mmo — CHANGES_REQUESTED** [review]

**Pull request merged**